docs/docs/09-rfcs/2023-03-08-wing-preview-environment.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Wing Cloud Preview Environments
+description: This document describes an MVP for the first Wing Cloud product - Wing Cloud Preview Environments.
+---
+
+# #3696 - Wing Cloud Preview Environments
+
+- **Author(s)**: @ainvoner, @skyrpex
+- **Submission Date**: 2023-08-03
+- **Stage**: Proposal
+- **Stage Date**: 2023-08-03
+
+This document describes an MVP for the first Wing Cloud product - Wing Cloud Preview Environments.
+
+## Introduction
+
+**Preview Environments** are the first feature of Wing Cloud. It will also be the first production grade product written in Wing language.
+
+## Requirements
+
+Wing Cloud users can build and deploy Wing applications to an ephemeral environment that uses the Wing Simulator platform, and use the Wing Console UI to interact with it.
+Once a preview environment is created, tests will be executed automatically and the results will be available in the preview environment dedicated comment in GitHub.
+Each preview environment will have a unique URL which enable developers to work together and share with external customers as needed.
+
+Wing Cloud will be integrated with GitHub.
+
+### Wing Cloud User Experience
+
+#### Sign-In to Wing Cloud (#1)
+
+1. On https://wing.cloud website click on sign in.
+2. Authenticate using your GitHub account.
+3. If you were already signed in, you will see the Wing Cloud dashboard (under https://wing.cloud directly).
```

Comment:
It make more sense to me to have `app.wing.cloud` or something like that.

Reply:
I understand there where already discussion about this, I am fine with what you decide

Reply:
I actually prefer that, too. It's technically simpler and will avoid us doing some black magic with CloudFront.